[E 250912 15:04:32 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getLtpData. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {'exchange': 'NSE', 'tradingsymbol': 'ICICIBANK-EQ', 'symboltoken': '4963'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getLtpData (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBBDF79D10>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:04:32 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getOrderBook. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getOrderBook?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBD7E91810>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:04:32 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getPosition. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getPosition?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBD7E91590>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:07:32 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getLtpData. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {'exchange': 'NSE', 'tradingsymbol': 'ICICIBANK-EQ', 'symboltoken': '4963'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getLtpData (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBBDF7BB10>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:07:32 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getOrderBook. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getOrderBook?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBD7E90F50>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:07:32 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getPosition. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getPosition?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBD7E907D0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:10:32 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getLtpData. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {'exchange': 'NSE', 'tradingsymbol': 'ICICIBANK-EQ', 'symboltoken': '4963'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getLtpData (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBBDF7BB10>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:10:32 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getOrderBook. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getOrderBook?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBBDF79A90>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:10:32 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getPosition. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getPosition?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBD6E06350>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:13:41 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getLtpData. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {'exchange': 'NSE', 'tradingsymbol': 'ICICIBANK-EQ', 'symboltoken': '4963'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getLtpData (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBBDF79A90>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:13:41 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getOrderBook. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getOrderBook?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBBDF7B9D0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:13:41 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getPosition. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getPosition?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBD6E06350>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:16:41 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getLtpData. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {'exchange': 'NSE', 'tradingsymbol': 'ICICIBANK-EQ', 'symboltoken': '4963'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getLtpData (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBBDF79D10>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:16:41 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getPosition. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getPosition?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBBDF79450>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250912 15:16:41 smartConnect:221] Error occurred while making a GET request to https://apiconnect.angelone.in/rest/secure/angelbroking/order/v1/getOrderBook. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:89:d2:13:87:8a', 'Accept': 'application/json', 'X-PrivateKey': 'Lp8Wcpst', 'X-UserType': 'USER', 'X-SourceID': 'WEB', 'Authorization': 'Bearer eyJhbGciOiJIUzUxMiJ9.eyJ1c2VybmFtZSI6IkE5MTk4MjIiLCJyb2xlcyI6MCwidXNlcnR5cGUiOiJVU0VSIiwidG9rZW4iOiJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKMWMyVnlYM1I1Y0dVaU9pSmpiR2xsYm5RaUxDSjBiMnRsYmw5MGVYQmxJam9pZEhKaFpHVmZZV05qWlhOelgzUnZhMlZ1SWl3aVoyMWZhV1FpT2pFekxDSnpiM1Z5WTJVaU9pSXpJaXdpWkdWMmFXTmxYMmxrSWpvaVpqY3dZVFJoWkdFdFkyTXdZaTB6WWpBeUxUbGhZVEF0WkRrME9XVTFOamhpTm1NeElpd2lhMmxrSWpvaWRISmhaR1ZmYTJWNVgzWXlJaXdpYjIxdVpXMWhibUZuWlhKcFpDSTZNVE1zSW5CeWIyUjFZM1J6SWpwN0ltUmxiV0YwSWpwN0luTjBZWFIxY3lJNkltRmpkR2wyWlNKOUxDSnRaaUk2ZXlKemRHRjBkWE1pT2lKaFkzUnBkbVVpZlgwc0ltbHpjeUk2SW5SeVlXUmxYMnh2WjJsdVgzTmxjblpwWTJVaUxDSnpkV0lpT2lKQk9URTVPREl5SWl3aVpYaHdJam94TnpVM056UTJOelUzTENKdVltWWlPakUzTlRjMk5qQXhOemNzSW1saGRDSTZNVGMxTnpZMk1ERTNOeXdpYW5ScElqb2lNalEzWlRobE9EVXRNVGsxTWkwME5URmxMV0pqWmpNdE1UTmlNak0xT0dNeFpUUTFJaXdpVkc5clpXNGlPaUlpZlEuc29SaFA1emduT3dYRnVoMzZobGp1cElIUjBUVlVqcWZJN3U2ang1RDY2UkJkZEZBemNTZFoyeUM1blRFOVBZRGY3OUUxdTdWTUQzTFZJMHdUSjRmc0s0ZTBKZjI1M0x1Nkl4a0RPV05IQ3I5elBzVUxiNVNEb1pVNGs2LVNOcThLZko5ckF4SVlXU1J1ZGxVby1EV0V2Vl9Qek5WZ1EwcVZ5Z2RBaXVnMEpnIiwiQVBJLUtFWSI6IkxwOFdjcHN0IiwiaWF0IjoxNzU3NjYwMzU3LCJleHAiOjE3NTc3MDE4MDB9.PTqLfyj3Dbd3pko6EhLTG0Tz_dpXNZ0R_TaC0myUnF0U6ClLJFBvVnOrnyAl46C4Ikj38HHDjpPrYLLhDEYsKA'}, Request: {}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/secure/angelbroking/order/v1/getOrderBook?%7B%7D (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CBD6E047D0>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
